cpims/logging_config.py

```diff
@@ -0,0 +1,59 @@
+from os import environ
+import logging.config
+import logging
+
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.logging import LoggingIntegration
+
+ENV_SENTRY_DNS = 'SENTRY_LOGGING_DNS'
+ENV_DEBUG = "CPIMS_DEBUG"
+ENV = "APP_ENV"
+
+APP_ENV_DEV = 'dev'
+SENTRY_DNS = environ.get(ENV_SENTRY_DNS)
```

Comment:
@SteveWaweru  You can create an account [here](https://sentry.io) and set this up in the env when deploying

Reply:
Alright. will create account for this